src/public/app/widgets/dialogs/add_link.js

```diff
@@ -121,7 +143,7 @@ export default class AddLinkDialog extends BasicWidget {
         };
 
         noteAutocompleteService.initNoteAutocomplete(this.$autoComplete, {
-            allowExternalLinks: true,
+            allowExternalLinks: false,
```

Comment:
@zadam when using the text editor control+L (i.e. using this dialog directly) I was never actually able to get an external link selected. Are we best off disabling it here, or should we fix external link selection inside the note selector?